ProductsWebApi/ProductsWebApi.Tests/UserServiceTest.cs

```diff
@@ -0,0 +1,100 @@
+﻿using AutoFixture;
+using FluentValidation;
+using Moq;
+using ProductsWebApi.Interface;
+using ProductsWebApi.Models;
+using ProductsWebApi.Services;
+using ProductsWebApi.Services.AuthenticationServices;
+using ProductsWebApi.Validator;
+using System.Linq;
+using Xunit;
+
+namespace ProductsWebApi.Tests
+{
+    public class UserServiceTest
+    {
+        [Theory]
+        [InlineData("")]
+        [InlineData(null)]
+        public void IsValidUser_IfUserNameIsNullOrEmpty_ThrowsValidationException(string userName)
+        {
+            var fixture = new Fixture();
+            var user = fixture.Build<Users>()
+                              .With(x => x.UserName, userName)
+                              .Create();
+
+            var userService = new UserService(Mock.Of<IUserRepository>(), new UserValidator());
```

Comment:
please, don't do this. 
Same changes here. move common lines to the constructor.

Reply:
Moved the common lines to the constructor in most of the test files.